### ISSUE TITLE
['Employment']: y contains previously unseen labels: 'Volunteer'

### ISSUE THREAD

**Issue by STHSF** [jpmml/sklearn2pmml#219]
HELP,  I add `invalid_value_treatment = "as_is"` when using GridSearch(),  but it also fails with a value error:

![image](https://user-images.githubusercontent.com/8037442/82201304-bd34f380-9932-11ea-812e-6cc0f3bfc75d.png)

**Comment by vruusmann** [comment]
Invalid value treatment "as is" means that the `CategoricalDomain` step is passing the "Volunteer" value forward to the `LabelEncoder` step.

It's the `sklearn.preprocessing.LabelEncoder` step that fails with the "y contains previously unseen labels" error. This is a standard Scikit-Learn class, and I can't do anything about it.

The fundamental problem is that the "Volunteer" label is a rare one, and in some cases `GridSearchCV` is splitting the dataset into training and testing subsets in a way that there are no "Volunteer" values in the training subset, but there is at least one in the testing subset.

A possible workaround is to manually move all "Volunteer" values from the testing subset back to the training subset.
